example/android/app/build.gradle

```diff
@@ -211,7 +211,7 @@ dependencies {
     implementation "com.facebook.react:react-native:+"  // From node_modules
     implementation "androidx.swiperefreshlayout:swiperefreshlayout:1.0.0"
     implementation 'com.google.android.material:material:1.4.0'
-    implementation 'com.stripe:stripe-wechatpay:17.1.0'
+    // implementation 'com.stripe:stripe-wechatpay:17.1.0' // UNCOMMENT
```

Comment:
temporary

Reply:
We probably shouldn't be shipping WeChat Pay anyway, it's disabled for the Mobile SDKs afaik